rosdep/python.yaml

```diff
@@ -1849,6 +1849,9 @@ python-sympy:
     raring: [python-sympy]
     saucy: [python-sympy]
     trusty: [python-sympy]
+python-tblib:
+  ubuntu: 
+    trusty: [python-tblib]
```

Comment:
Only adding it for Trusty won't allow you to release any package. Please fill in additional Ubuntu versions.